server/src/pages/shot/controller.js

```diff
@@ -20,6 +20,36 @@ function shouldHighlightEditIcon(model) {
   return !hasSeen;
 }
 
+function shouldShowPromo(model) {
+  if (!model.isOwner || !model.enableAnnotations) {
+    return false;
+  }
+  let show = false;
+  const count = localStorage.hasSeenPromoDialog;
+  const interval = 1; // in minutes
```

Comment:
Now that we have increased frequency of promo display to 3, interval here controls when the next notification should be shown from the last seen , it's set to 1 minute for now